packages/fxa-shared/subscriptions/types.ts

```diff
@@ -118,6 +118,8 @@ export type WebSubscription = Pick<
     >;
     subscription_id: Stripe.Subscription['id'];
     promotion_code?: string;
+    promotion_duration: string | null;
```

Comment:
Not that this is necessarily part of this review but how is the `promotion_code` string or undefined when it is similarly string or else where? I may be incorrect but was just curious about that inconsistency. Don't feel the need to dig around for the answer, more curious if you have it off the top of your head.